tests/e2e/pact/services/prepare-example-fetcher.ts

```diff
@@ -0,0 +1,35 @@
+import path from "path"
+import { PrepareCase } from "../models/cases/prepare-case"
+import { exampleFiles } from "./example-files-fetcher"
+
+const examplesRootPath = "../resources/parent-prescription"
+
+const prepareResponseFiles = exampleFiles.filter(exampleFile => exampleFile.isResponse && exampleFile.endpoint === "prepare")
+
+const prepareRequestFiles = exampleFiles.filter(exampleFile =>
```

Comment:
Won't comment twice, but this file needs the same changes as `convert-example-fetcher`